api/types/events/events.pb.go

```diff
@@ -1655,7 +1655,9 @@ func (m *AccessRequestCreate) XXX_DiscardUnknown() {
 
 var xxx_messageInfo_AccessRequestCreate proto.InternalMessageInfo
 
-// ResourceID is a unique identifier for a teleport resource.
+// ResourceID is a unique identifier for a teleport resource. This is duplicated
+// from api/types/types.proto to decouple the api and events types and because
+// neither file currently imports the other.
```

Comment:
this got in after rebasing master, double checked that this wasn't in master (it looks like we put extra comment in `events.proto` but forgot to run `make grpc` )